### PR TITLE
DDF-1740 added inResponseTo validation to SAML logout

### PR DESCRIPTION
**Pull request by ryeats**
@jckilmer @roelens8 @coyotesqrl @tbatie @wbarnie @rzwiefel

### CODE REVIEW

File: platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java [hunk 1]
```diff
@@ -143,6 +143,12 @@ public LogoutRequest buildLogoutRequest(@NotNull String nameIdString,
                 id);
     }
 
+    /**
+     *  @apiNote ONLY use this method in error cases when createing a valid response pass the inResponseTo id
```
Comment (tbatie) [inline platform/security/core/security-core-impl/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java]: creating*

**Comment by tbatie** [comment]
LGTM :+1:

**Comment by roelens8** [comment]
:+1:

**Comment by coyotesqrl** [comment]
:+1:

**Pull request merged**